pyshtools/shclasses/shcoeffs.py

```diff
@@ -4008,12 +4010,11 @@ def _expandGLQ(self, zeros, lmax, lmax_calc, extend):
                 "'unnorm'. Input value is {:s}."
                 .format(repr(self.normalization)))
 
-        if zeros is None:
+        if backend_name() != "ducc" and zeros is None:
```

Comment:
`== 'shtools'`? I'm not insisting, just not sure what is the best for the future.